browser/scripts/application.js

```diff
@@ -1810,6 +1846,26 @@ Application.prototype.start = function() {
 	this.midPane = new E2.MidPane()
 }
 
+/**
+ * Update the graph title in the header bar. Make sure the title fits within the space between file selection and play/login buttons.
+ * @param  {String} title  Title to use
+ */
+Application.prototype.updateGraphTitle = function(title) {
+
+	// Commented out for now
```

Comment:
I'd like not to have completely commented out code paths in the application. Better to just remove the updateGraphTitle function and references?